app.test.js

```diff
@@ -1,5 +1,22 @@
-import { describe, expect, it } from 'vitest';
+import {
+  afterAll,
+  beforeAll, beforeEach, describe, expect, it,
+} from 'vitest';
+import { client } from './connections.js';
 import app from './app.js';
+import { dbName, dbCollection, addNewSoldier } from './database.js';
+
+beforeAll(async () => {
+  await client.connect();
+});
+
+afterAll(async () => {
+  await client.close();
+});
+
+beforeEach(async () => {
+  await client.db(dbName).collection(dbCollection).deleteMany({});
```

Comment:
you can [`drop`](https://mongodb.github.io/node-mongodb-native/5.0/classes/Collection.html#drop) the collection if you like

Reply:
is it preferred? or is cleaning fine as well?